boards/arm/mimxrt685_evk/pinmux.c

```diff
@@ -281,4 +281,5 @@ static int mimxrt685_evk_pinmux_init(const struct device *dev)
 	return 0;
 }
 
-SYS_INIT(mimxrt685_evk_pinmux_init, PRE_KERNEL_1, CONFIG_PINMUX_INIT_PRIORITY);
+/* priority set to CONFIG_PINMUX_INIT_PRIORITY value */
+SYS_INIT(mimxrt685_evk_pinmux_init, PRE_KERNEL_1, 45);
```

Comment:
Why are you changing this to be hard-coded?

Reply:
Because the RT685 doesn't actually use the pinmux API / implement a pinmux driver.  So CONFIG_PINMUX doesn't make sense to set for it.  Once CONFIG_PINMUX goes away, than CONFIG_PINMUX_INIT_PRIORITY isn't available either.